velox/functions/lib/string/StringCore.h

```diff
@@ -43,6 +43,29 @@
 #endif
 
 namespace facebook::velox::functions {
+namespace detail {
+
+// Helper function to check if a character is cased. Compatible with the isCased
+// function of jdk used by 'toLowerCase' in Spark SQL.
+FOLLY_ALWAYS_INLINE bool isCased(int ch) {
```

Comment:
Could you add a comment to mark that this logic comes from the `isCased` method in `ConditionalSpecialCasting.java` of the JDK? This makes it easier to trace the source of the code logic.

Reply:
Updated, thanks for the suggestion.